src/services/Mailer.services.js

```diff
@@ -0,0 +1,101 @@
+import pug from 'pug';
+import sgMail from '@sendgrid/mail';
+import config from '../config';
+
+sgMail.setApiKey(config.SENDGRID_API_KEY);
+
+/**
+* Class representing a mailing services.
+*/
+export default class mailer {
```

Comment:
I think we should avoid classes and static methods and defer to objects:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Details_of_the_Object_Model

Reply:
I'm using classes because I need a constructor there. So as to pass data to non-static classes